Tests.Integration.Presentation.Web/Tools/V2/OrganizationV2Helper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Presentation.Web.Models.External.V2.Response;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Tools.V2
+{
+    public static class OrganizationV2Helper
+    {
+        public static async Task<IEnumerable<OrganizationResponseDTO>> GetOrganizationsForWhichUserIsRightsHolder(Cookie loginCookie, int page = 0, int pageSize = 10)
+        {
+            using var response = await HttpApi.GetWithCookieAsync(TestEnvironment.CreateUrl($"api/v2/organizations/with-rightsholder-access?page={page}&pageSize={pageSize}"), loginCookie);
```

Comment:
Burde V2 ikke testes med Api Token egentlig?

Reply:
I princippet er det ikke et krav men jo... lad os ændre det til det. Så viser vi den rigtige vej for dem der skal bruge det. Jeg retter det til.